web/configmeter.html

```diff
@@ -25,6 +25,9 @@
                     <a class="nav-link" href="/config-web">Web</a>
                 </li>
                 <li class="nav-item">
+                    <a class="nav-link " href="/config-domoticz">Domoticz</a>
+                </li>
+               <li class="nav-item">
                     <a class="nav-link" href="/config-system">System</a>
```

Comment:
Just thinking, maybe move this link into MQTT configuration and only show if Domoticz is selected?

Reply:
agree, see above.  May need help for this one.

Reply:
OK no problem, commit whatever you can and I'll make final adjustments

Reply:
No changes now. I leave it to you to make the adjustments on menues that you think is right.